src/Credentials.js

```diff
@@ -192,6 +197,23 @@ class Credentials {
     return this.requestDisclosure(params)
   }
 
+/**
```

Comment:
@zachferland we need to mark this as deprecated and remember to remove it from the major libraries